xinference/model/llm/llm_family.json

```diff
@@ -7749,6 +7749,99 @@
       "<|im_end|>"
     ]
   },
+  {
+    "version": 1,
+    "context_length": 32768,
+    "model_name": "HuatuoGPT-o1-Qwen2.5",
+    "model_lang": [
+      "en",
+      "zh"
+    ],
+    "model_ability": [
+      "chat",
+      "tools"
+    ],
+    "model_description": "HuatuoGPT-o1 is a medical LLM designed for advanced medical reasoning. It generates a complex thought process, reflecting and refining its reasoning, before providing a final response.",
+    "model_specs": [
+      {
+        "model_format": "pytorch",
+        "model_size_in_billions": 7,
+        "quantizations": [
+          "4-bit",
```

Comment:
The quantization for PyTorch has been removed, the merged model Xiyan needs to be modified as well, you can modify it in this PR